lib/playground.dart

```diff
@@ -278,17 +281,24 @@ class Playground implements GistContainer, GistController {
     _editpanel.children.first.attributes['flex'] = '';
     editor.resize();
 
-    keys.bind(['ctrl-s'], _handleSave);
-    keys.bind(['ctrl-enter'], _handleRun);
+    keys.bind(['ctrl-s'], sharingDialog.show, "Share");
```

Comment:
I suspect we want this hidden somehow. It's nice to track when the hit ctrl-s, but not critical. Let's comment out the key-binding for now.